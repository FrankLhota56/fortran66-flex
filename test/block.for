C                                                                       BLCK0010
C Data test                                                             BLCK0020
C                                                                       BLCK0030
C Also tests 72 character limit                                         BLCK0040
C                                                                       BLCK0050
      DoublePrecision Z axis                                            BLCK0060
      BLOCK Data Init                                                   BLCK0070
     0data Greet / 66 hNow is the time for all good men to come to the aBLCK0080
     1id of their party /                                               BLCK0090
      data i / 666 /                                                    BLCK0100
      Da Ta X, Y,Z / 1.3, 4.5E+1, 10.66E-2 /, T1 / 5D-1/                BLCK0110
      End                                                               BLCK0120
