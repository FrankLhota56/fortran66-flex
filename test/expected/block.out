Line     6: DOUBLE
Line     6: PRECISION
Line     6: ID(zaxis)
Line     7: EOS
Line     7: BLOCK
Line     7: DATA
Line     7: ID(init)
Line     8: EOS
Line     8: DATA
Line     8: ID(greet)
Line     8: '/'
Line     9: HOLLERITH_LIT(Now is the time for all good men to come to the aid of their party)
Line     9: '/'
Line    10: EOS
Line    10: DATA
Line    10: ID(i)
Line    10: '/'
Line    10: INT_LIT(666)
Line    10: '/'
Line    11: EOS
Line    11: DATA
Line    11: ID(x)
Line    11: ','
Line    11: ID(y)
Line    11: ','
Line    11: ID(z)
Line    11: '/'
Line    11: REAL_LIT(1.300000)
Line    11: ','
Line    11: REAL_LIT(45.000000)
Line    11: ','
Line    11: REAL_LIT(0.106600)
Line    11: '/'
Line    11: ','
Line    11: ID(t1)
Line    11: '/'
Line    11: DBL_PREC_LIT(0.500000)
Line    11: '/'
Line    12: EOS
Line    12: END
